job-dsl-core/src/main/docs/examples/javaposse/jobdsl/dsl/helpers/triggers/TriggerContext/pullRequest.groovy

```diff
@@ -24,6 +24,7 @@ job('example') {
                 commitStatus {
                     context('deploy to staging site')
                     startedStatus('deploying to staging site...')
+                    statusUrl('--none--')
```

Comment:
--none-- vs. null is an issue of the ghprb plugin, use a real URL in the example